### PR TITLE
Add package feature1

### PR DESCRIPTION
**Pull request by Santhoshkumar-b**

### CODE REVIEW

File: src/main/java/in/santhosh/model/TourPackageDetails.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package in.santhosh.model;
+
+import java.time.LocalDate;
+
+public class TourPackageDetails {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/santhosh/model/TourPackageDetails.java:5]: Remove 's'. 
TourPackageDetail

File: src/main/java/in/santhosh/servlet/PackageAction.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import in.santhosh.model.TourPackageDetails;
+import in.santhosh.service.Packages;
+
+/**
+ * Servlet implementation class PackageAction
+ */
+@WebServlet("/PackageAction")
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/santhosh/servlet/PackageAction.java:18]: Add/Update/Delete/List

File: src/main/webapp/AddPackage.jsp [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+		}
+		%>
+		<%
+		String infoMessage = request.getParameter("infoMessage");
+		if(infoMessage != null){
+			out.println("<font color='green'>" + infoMessage + "</font>");
+		}
+		%>
+		<form action="PackageAction">
+		<label for="countryName"><strong>Country name</strong></label><br>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/AddPackage.jsp:23]: Select option country name.

File: src/main/webapp/AddPackage.jsp [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+		<br>
+		<label for="days"><strong>Days</strong></label><br>
+		<input type="number" name="days" placeholder="Enter the number of days" min=3 max=15 required>
+		<br>
+		<label for="startDate"><strong>Journey start date</strong></label><br>
+		<input type="date" name="startDate" placeholder="Journey start date" required>
+		<br>
+		<label for="endDate"><strong>Journey end date</strong></label><br>
+		<input type="date"  name="endDate" placeholder="Journey end date" required>
+		<br>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/AddPackage.jsp:37]: <br/> - Self closing tag.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**